files/apt.sh

```diff
@@ -18,9 +18,9 @@ apt_check_latest() {
   candidate="${candidate#*: }"
 
   if [[ $installed != $candidate ]]; then
-    cmd_status="$(apt_status "{ \"status\":\"\${Status}\", \"version\":\"${installed}\", \"latest\":\"${candidate}\" }")"
+    cmd_status="$(apt_status "{ \"status\":\"\${db:Status-Status}\", \"version\":\"${installed}\", \"latest\":\"${candidate}\" }")"
```

Comment:
We can also hardcode `installed` here and bellow if you prefer, because it is what we are going to get all the times here and it makes the code slightly more explicit.

Reply:
oh good idea, I like thar